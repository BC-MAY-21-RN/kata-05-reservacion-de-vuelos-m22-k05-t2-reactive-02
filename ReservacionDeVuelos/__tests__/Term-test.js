import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import Terms from '../src/components/atoms/Term';

describe('Tests Terms component', () => {
  it('Terms snapshot', () => {
    const renderTerms = renderer
      .create(
        <Terms
          changeForm={null}
          ischecked={false}
          isfirst={false}
          term={''}
          text={''}
        />,
      )
      .toJSON();
    expect(renderTerms).toMatchSnapshot();
  });
});
