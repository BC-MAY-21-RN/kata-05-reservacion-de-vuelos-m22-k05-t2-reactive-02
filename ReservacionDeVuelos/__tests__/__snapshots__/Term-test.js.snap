// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests Terms component Terms snapshot 1`] = `
<View
  style={
    Object {
      "height": 20.1,
      "marginBottom": 10,
      "marginLeft": 20,
      "marginTop": 0,
      "width": 710,
    }
  }
>
  <View
    style={
      Object {
        "flex": 1,
        "flexDirection": "row",
      }
    }
  >
    <View
      accessible={true}
      collapsable={false}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "borderColor": "#9CA5B4",
          "borderWidth": 1,
          "height": 20,
          "justifyContent": "center",
          "opacity": 1,
          "width": 20,
        }
      }
    />
    <Text
      style={
        Object {
          "color": "#9CA5B4",
          "fontSize": 16,
          "marginLeft": 10,
        }
      }
    >
      
    </Text>
  </View>
</View>
`;
